aivalidation_dictionary.txt

```diff
@@ -94,3 +95,6 @@ webex
 webserver
 Weijs
 Woudloper
+ransparency
```

Comment:
@berrydenhartog what do you think about this? Or should I remove the capitalized bolded text in the project description itself?